kafka-ui-react-app/src/components/Topics/Topic/Details/Details.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { ClusterName, TopicName } from 'redux/interfaces';
-import { Topic, TopicDetails } from 'generated-sources';
+import { ClusterFeaturesEnum, Topic, TopicDetails } from 'generated-sources';
 import { NavLink, Switch, Route, Link, useHistory } from 'react-router-dom';
+import { useSelector } from 'react-redux';
```

Comment:
There's the DetailsContainer component, which is responsible for providing the data from redux to the component. Better use it for the sake of consistency